internal/config/projectconfig/apply.go

```diff
@@ -0,0 +1,43 @@
+package projectconfig
```

Comment:
I don't think this is the right place for this. Maybe move to an `internal/apply` package?